views/imagestream-listing.html

```diff
@@ -1,4 +1,4 @@
-<table class="listing-ct">
+<table ng-class="classNames ? '{{classNames}}' : 'listing-ct'">
```

Comment:
I think a lot of other styles in here that you probably want to keep reference listing-ct. Either those need to change, or this should keep listing-ct even when classNames is present.